docs/Manual/07 Plans - Vectorization and Parallelization.md

```diff
@@ -140,7 +140,7 @@ plan.vectorize(j)
 ```
 Applying this sequence of instructions allows the compiler to automatically create an optimized kernel from loops `i, k, j`.
 
-For simplicity, let's assume that the matrix sizes, defined by M, N, S are M=3, N=4, S=2.
+For simplicity, assume that the matrix sizes defined by M, N, and S are 3, 4, and 2.
```

Comment:
maybe "3, 4, and 2 respectively."

Reply:
Yup, I had used it before. But then, without it made more sense back then. Fixing as per your feedback!